arbnode/api.go

```diff
@@ -142,8 +159,8 @@ func (api *ArbDebugAPI) PricingModel(ctx context.Context, start, end rpc.BlockNu
 		L1LastUpdateTime:     make([]uint64, blocks),
 	}
 
-	for i := uint64(0); i < uint64(blocks); i++ {
-		state, header, err := stateAndHeader(api.blockchain, i+uint64(start))
+	for i := uint64(0); i < uint64(blocks); i += 1 {
```

Comment:
I'm assuming the increment was changed in a previous revision but it could changed back to `i++` now

Reply:
Ah yeah that's what happened. Reverted :)